pages/product.html

```diff
@@ -206,147 +253,191 @@ <h6 class="text-uppercase mt-2p5">Sort By</h6>
             </label>
           </div>
+          <button class="button primary radius-0 add-filter-btn mt-2">
+            Add Filters
+          </button>
         </aside>
         <div class="grid-container border-left-1 p-2">
           <div class="card-vertical m-1">
-            <div class="badge">
-              <img
-                class="card-img-vertical"
-                src="/images/fancy-cup.jpg"
-                alt="product"
-              />
-              <span class="badge-text badge-on-left success px-0p5"
-                >Bestseller</span
-              >
-              <span><i class="far fa-heart wishlist-btn wishlisted"></i></span>
-            </div>
-            <div class="card-details text-center">
-              <div class="title">Fancy Cup with Gold</div>
-              <div class="card-footer-vertical">
-                <span class="discountedPrice">Rs. 499</span>
-                <span class="originalPrice pl-0p5">Rs. 999</span>
+            <a
+              href="/pages/singleproduct.html"
+              class="no-underline primary-text"
+            >
+              <div class="badge">
+                <img
+                  class="card-img-vertical"
+                  src="/images/fancy-cup.jpg"
+                  alt="product"
+                />
+                <span class="badge-text badge-on-left success px-0p5"
+                  >Bestseller</span
+                >
+                <span
+                  ><i class="far fa-heart wishlist-btn wishlisted"></i
+                ></span>
               </div>
-            </div>
+              <div class="card-details text-center">
+                <div class="title">Fancy Cup with Gold</div>
+                <div class="card-footer-vertical">
+                  <span class="discountedPrice">Rs. 499</span>
+                  <span class="originalPrice pl-0p5">Rs. 999</span>
+                </div>
+              </div>
+            </a>
             <button class="plain-button card-button mt-0p5">Add to Cart</button>
           </div>
 
           <div class="card-vertical m-1">
-            <div class="badge">
-              <img
-                class="card-img-vertical"
-                src="/images/jug-planter.jpg"
-                alt="product"
-              />
-              <span><i class="far fa-heart wishlist-btn"></i></span>
-            </div>
-            <div class="card-details text-center">
-              <div class="title">White Jug Planter</div>
-              <div class="card-footer-vertical">
-                <span class="discountedPrice">Rs. 499</span>
-                <span class="originalPrice pl-0p5">Rs. 999</span>
+            <a
+              href="/pages/singleproduct.html"
+              class="no-underline primary-text"
+            >
+              <div class="badge">
+                <img
+                  class="card-img-vertical"
+                  src="/images/jug-planter.jpg"
+                  alt="product"
```

Comment:
Suggestion: In all your image you have assigned the product to all alt values. It would be nice if you can give more meaningful value to them. 
For - e.g: if the product image is of jug planter you can give ```alt =" jug planter```

Reply:
Yes, I will. Due to time crunch, I gave generic names. I will update it later. Thanks